yellowbrick/classifier/prcurve.py

```diff
@@ -79,6 +80,14 @@ class PrecisionRecallCurve(ClassificationScoreVisualizer):
         field to filter the visualization for specific classes. For more advanced
         usage specify an encoder rather than class labels.
 
+    color : optional list or tuple of colors to colorize the curves when
+        `per_class=True`, default: None. Use either color to colorize the curves
+        on a per class basis or colormap to color them on a continuous scale.
+
+    colormap : optional string or matplotlib cmap to colorize curves, default: None
+        Use either color to colorize the curves on a per class basis or
+        colormap to color them on a continuous scale.
```

Comment:
Thank you for updating the docstring here @VladSkripniuk! I propose that we provide a little bit additional information so that users will understand the behavior of these parameters (as well as some of the mutually exclusive conditions), e.g.:

```
colors : list of strings,  default: None
    An optional list or tuple of colors to colorize the curves when 
    ``per_class=True``. If ``per_class=False``, this parameter will
    be ignored. If both ``colors`` and ``colormap`` are provided,
    ``colormap`` will be ignored.

colormap : string or Matplotlib colormap, default: None
    An optional string or Matplotlib colormap to colorize the curves
    when ``per_class=True``. If ``per_class=False``, this parameter
    will be ignored. If both ``colors`` and ``colormap`` are provided,
    ``colormap`` will be ignored.
```

Reply:
That's a good point!